packages/mux-player/src/index.ts

```diff
@@ -135,6 +136,16 @@ function getThemeTemplate(el: MuxPlayerElement) {
   }
 }
 
+function getHideDuration(el: MuxPlayerElement) {
```

Comment:
thought(non-blocking): Given these acrobatics, I'm wondering if on the Media Chrome side we should break up the display for duration vs. current/remaining time into separate nodes with their own css var for display. That way we can avoid these acrobatics in player.

Reply:
yes, it's worth exploring again. I have done this in the past but for some reason dropped that effort